pillar/spoonbill.sls

```diff
@@ -0,0 +1,22 @@
+docker:
+  user: deployer
+  uid: 1002
+  docker_compose:
+    version: 1.29.2
+
+docker_apps:
+  spoonbill:
+    target: spoonbill
+    env:
+      DOMAIN: &DOMAIN flatten.open-contracting.org
+      API_DOMAIN: *DOMAIN
+      TRAEFIK_IP: 65.21.93.141
+      ACME_EMAIL: shakh@quintagroup.org
```

Comment:
Please create an issue to change this email address to sysadmin@open-contracting.org (might require some other change with Let's Encrypt - I haven't looked into it)

Reply:
https://github.com/open-contracting/deploy/issues/349